app/models/reservation.rb

```diff
@@ -121,7 +121,15 @@ def self.relay_in_progress
     where("actual_start_at IS NOT NULL AND actual_end_at IS NULL")
   end
 
-  scope :not_offline, -> { where(type: nil) }
+  def self.upcoming_offline(start_at_limit)
```

Comment:
This is borrowed from #705. Both PRs need it, and as it turns out they needed different `start_at_limit` values so it made sense to do away with the default limit.